inference-engine/src/multi_device/multi_device_exec_network.cpp

```diff
@@ -582,6 +584,10 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetConfig(const std::st
 
 InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetMetric(const std::string &name) const {
     if (_workModeIsAUTO) {
+        if (name == METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS)) {
+            unsigned int res = 4u;
```

Comment:
we discussed 8 on the meeting, I will try to find better solution (that will query some device info) anyway

Reply:
at least it is better to report max(4, MetricValueFromFirstLoadedDevice), so that if the accelerator (KMB or GPU) loaded first (from the cache) we don't override the val

Reply:
use max(8, MetricValueFromFirstLoadedDevice)